app-frontend/src/app/pages/error/error.controller.js

```diff
@@ -0,0 +1,29 @@
+class ErrorController {
+    constructor( // eslint-disable-line max-params
+        $log, $window, $state, APP_CONFIG, $interval, config
+    ) {
+        'ngInject';
+
+        this.$log = $log;
+        this.config = config;
+        this.$state = $state;
+        this.$window = $window;
+
+        if (!APP_CONFIG.error) {
+            $state.go('browse');
+        }
+        $log.log('Error while loading app config from /config');
+        this.count = 0;
+        $interval(this.getConfig.bind(this), 5000);
+    }
+
+    getConfig() {
+        this.count = this.count + 1;
+        this.$log.log('Trying to load config: try # ', this.count);
```

Comment:
I think a lot of these log statements should be removed, or use `debug` rather than `log`.